app/services/coronavirus/pages/sub_section_processor.rb

```diff
@@ -4,18 +4,20 @@ def self.call(*args)
       new(*args).output
     end
 
-    attr_reader :sub_sections, :featured_link
+    attr_reader :sub_sections, :action_link
 
     def initialize(sub_sections)
       @sub_sections = [sub_sections].flatten
-      @featured_link = nil
+      @action_link = { url: nil, content: nil, summary: nil }.with_indifferent_access
```

Comment:
I'd have thought this could be initialised to an empty hash since accessing the keys will return nil anyway, I'm not sure if anything is making use of indifferent_access so I expect that can be removed.